packages/time/src/types.d.ts

```diff
@@ -205,6 +230,24 @@ export interface TimerRepeater {
   disable: () => void;
 }
 
+/**
+ * TimeMath supports simple arithmetic on typed Time values, enforcing that
+ * values are combined in type-compatible ways. You can add 3 minutes to 3pm,
+ * or 5 minutes to a half hour, but it makes no sense to add 3pm and 5pm.
+ * Subtracting two Timestamps does produce a useful difference.
+ *
+ * The types also prevent you from accidentally combining time values from
+ * different TimerServices. If some chains track time in blocks, while others
+ * follow wall clock time, using the correct types means you don't have to worry
+ * about timezones or how time is represented on a particular chain. This also
```

Comment:
Did you mean "using the correct brands" rather than "using the correct types"? It's not clear to me how using the correct types means I don't have to worry about the rest. Perhaps it means I'll be sure to get early errors if I do it wrong, but "don't have to worry about it" seems much stronger than that.

Reply:
You are correct. Getting the types right means you won't be adding 3pm and 5pm, but getting the brands right means you won't be subtracting 20 blocks from 10am.